sublime-package.json

```diff
@@ -739,6 +739,12 @@
               "default": false,
               "markdownDescription": "Show inlay hints in the editor. Inlay hints are short annotations within the code, which show variable types or parameter names.\nThis is the default value used for new windows but can be overriden per-window using the `LSP: Toggle Inlay Hints` command from the Command Palette, Main Menu or a custom keybinding."
             },
+            "inlay_hints_max_length": {
+              "type": "integer",
+              "default": 30,
+              "minimum": 0,
+              "markdownDescription": "Maximum length for inlay hints. Truncates exceeding characters and adds an ellipsis."
```

Comment:
0 means "no truncation", right? We should specify that.

Reply:
Done in [572b1cd](https://github.com/sublimelsp/LSP/pull/2514/commits/572b1cd7699544fc92a1deedcdd550d190254818)